ctapipe/instrument/InstrumentDescription.py

```diff
@@ -905,6 +907,222 @@ def load_config(filename):
     print('Astropy tables have been created.')
     return(telescope,camera,optics)
 
+
+def override_in_table(table,item={}):
+    # Load by default
+    try:
+        newvalue=item[newvalue]
+    except:
+        newvalue=None
+    try:
+        variable=item[variable]
+    except:
+        variable=None
+    try:
+        entry=item[entry]
+    except:
+        entry = None
+    
+
+    # Case A) no variable, no item, override the whole table
+
+    if newvalue==None:
+        return(table)
```

Comment:
Return became a function in Python 3, the same applies to python. Somehow became also possible to use them as functions in recent version of the Python 2.X branch and I adopted it to try to make the code as compliant as possible with 3.X. But it's fine to go back to the old 2.X behavior, I will change it.